Gui/opensim/build.xml

```diff
@@ -32,12 +32,28 @@
     <condition property="opensim.dist.dir" value="dist/OpenSim.app/Contents/Resources/OpenSim">
         <os family="mac"/>
     </condition>
+    <condition property="opensim.dist.dir" value="dist/installer/OpenSim">
+        <and>
+            <os family="unix"/>
+            <not>
+                <os family="mac"/>
+            </not>
+        </and>
+    </condition>
     <condition property="opensim.threejs.dir" value="${opensim.dist.dir}/threejs">
         <os family="mac"/>
     </condition>
     <condition property="opensim.threejs.dir" value="${opensim.dist.dir}/OpenSim/threejs">
         <os family="windows"/>
     </condition>
+    <condition property="opensim.threejs.dir" value="${opensim.dist.dir}/threejs">
+        <and>
+            <os family="unix"/>
+            <not>
+                <os family="mac"/>
+            </not>
+        </and>
+    </condition>
```

Comment:
This is the same as the definition for `mac`; could you instead change line 44 to `<os family="unix"/>`?